app/src/components/pages/ProfessorsList.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { DataGrid, GridColDef } from '@mui/x-data-grid';
+import { parseCalendarTeacher } from 'utils/utils';
+import classData from 'data/clean.json';
+import { Box } from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+import { Search, SearchIconWrapper, StyledInputBase } from 'components/styles/styles';
+import { useState } from 'react';
+import { ICalendarTeacherItem } from 'components/shared/interfaces/timetable.interfaces';
+import Link from '@mui/material/Link';
+
+const columns: GridColDef[] = [
+  { field: 'id', headerName: 'ID', width: 70 },
+  { field: 'teacherName', headerName: 'Username', width: 130 },
+  {
+    field: 'link',
+    headerName: 'View Professor Profile',
+    width: 200,
+    renderCell: (params: any) => <Link href={params.value}> View profile </Link>

```

Comment:
Work on it

Reply:
I think we should pass the teacher id ？The teacher username might be same when they teach same course (or same section), so it will be duplicated in the professor list view.